enterprise/redhat/openshift-ai/gaudi/docker/requirements.txt

```diff
@@ -9,22 +9,22 @@ plotly~=5.24.1
 scikit-learn
 scipy~=1.14.1
 skl2onnx~=1.17.0
-codeflare-sdk~=0.21.1
+codeflare-sdk~=0.23.1
 
 # DB connectors
 pymongo~=4.10.1
 psycopg~=3.2.3
-pyodbc~=5.1.0
+pyodbc~=5.2.0
 mysql-connector-python~=9.1.0
 
 # JupyterLab packages
-odh-elyra~=4.0.3
-jupyterlab~=4.2.5 # Wait on upgrade till plugins are ready
+odh-elyra~=4.2.0
```

Comment:
`odh-elyra` version is latest & it depends on jupyterlab 4.2.5. containers builds failing due to this reason. at this time, I will keep the PR open for now